ksql-engine/src/test/java/io/confluent/ksql/integration/JoinIntTest.java

```diff
@@ -78,10 +94,10 @@ public void shouldLeftJoinOrderAndItems() throws Exception {
     final String testStreamName = "OrderedWithDescription".toUpperCase();
 
     final String queryString = String.format(
-            "CREATE STREAM %s AS SELECT ORDERID, ITEMID, ORDERUNITS, DESCRIPTION FROM orders LEFT JOIN items " +
-                    " on orders.ITEMID = items.ID WHERE orders.ITEMID = 'ITEM_1' ;",
-            testStreamName
-    );
+            "CREATE STREAM %s AS SELECT ORDERID, ITEMID, ORDERUNITS, DESCRIPTION FROM %s LEFT JOIN"
+            + " %s " +
```

Comment:
formatting is a bit strange

Reply:
formatting is still a bit strange.

Reply:
Formatting is still a bit strange :( The arguments should line up with the string literal. Also the string is needless split over 3 lines -- it could fit into two.

Reply:
If you are using Intellij, try setting the google-style.xml in your settings. Then re-indent the region. it _should_ just do the right thing.